EventHubs/tests/baseeventhubtest.py

```diff
@@ -49,39 +48,48 @@ def wait_for_table_results(self, query):
             sleep(15)
             max_retries -= 1
 
-    def get_table_service(self):
-        storage_client = StorageManagementClient(self.credentials,
-                                                 self.subscription_id)
-        STORAGE_ACCOUNT_NAME = self.get_resource_name(self.STORAGE_ACCOUNT_NAME, "Microsoft.Storage/storageAccounts")
-        storage_keys = storage_client.storage_accounts.list_keys(
-            self.RESOURCE_GROUP_NAME, STORAGE_ACCOUNT_NAME)
-        acckey = storage_keys.keys[0].value
-        table_service = TableService(account_name=STORAGE_ACCOUNT_NAME,
-                                     account_key=acckey)
-        return table_service
-
     def insert_mock_logs_in_EventHub(self, filename):
         print("Inserting fake logs in EventHub")
+        
```

Comment:
this function should not have been changed revert it back to older indentation

Reply:
ok